.github/dependabot.yml

```diff
@@ -3,35 +3,44 @@ updates:
 - package-ecosystem: composer
   directory: "/"
   schedule:
-    interval: daily
+    interval: weekly
```

Comment:
How come this line gets indented? In some scenarios there is the "-"?

Reply:
Rob covered the explanation here https://hmn.slack.com/archives/C5WSNGC9F/p1636710824228000